packages/main/src/FileUploaderPopover.hbs

```diff
@@ -9,7 +9,9 @@
 	horizontal-align="Left"
 >
 	<div slot="header" class="{{classes.popoverValueState}}" style="{{styles.popoverHeader}}">
-		<ui5-icon class="ui5-input-value-state-message-icon" name="{{_valueStateMessageInputIcon}}"></ui5-icon>
+		{{#if _valueStateMessageInputIcon}}
```

Comment:
This private method has wrong code documentation. It is not only relevant for the horizon theme so it seems.

Reply:
Yes, It doesn't work only on horizon theme, but the difference made from the result can be observed only on horizon theme, as the value state icon is only visible on the horizon theme. So it is and it is not at the same time.